tools/gui/plotter/plotter_backend.py

```diff
@@ -0,0 +1,62 @@
+# This should probably go with the rest of the code for the plotter tab
+import os
+
+import pandas
+import matplotlib
+matplotlib.use('tkagg')
+import matplotlib.pyplot as plt
+
+matplotlib.rc('font', **{'sans-serif' : 'Arial',
+                         'family' : 'sans-serif'})
+
+def get_csv_headers(filename):
+    """
+    Returns a list of a csv files column names. Meant to generate choices for drop down menu.
+
+    Parameters
+    ----------
+    filename: str
+        Filename of csv file to be parsed
+
+    Returns
+    -------
+    list: List of column names from filename
+    """
+
+    return pandas.read_csv(filename, encoding = "utf-8").columns.tolist()
+
+
+def submit_chosen_columns(filename, column1, column2):
```

Comment:
Use more descriptive names here
```suggestion
def submit_chosen_columns(filename, x_axis_label, y_axis_label):
```